app/resources/css-default.template

```diff
@@ -282,38 +280,48 @@ We don't use "display: table" because IE 6/7 doesn't support it. IE 6/7 is
 over 10% in penetration ratio.
 */
 
-.fields-table {
-  padding: 4px;
+.card {
+  background: #ffffff;
+  border: 1px solid #dadada;
+  box-shadow: 2px 2px 3px 0px #a0a0a0;
+  padding: 10px;
+  margin: 10px;
+  border-radius: 3px;
+}
+
+.card h1 {
+  margin: auto;
```

Comment:
What does this mean? Maybe it was necessary to centerize it? Can you remove it if it's no longer necessary?